core/modules/wbase/Task.h

```diff
@@ -178,10 +182,12 @@ class Task : public util::CommandForThreadPool {
     std::chrono::milliseconds finished(std::chrono::system_clock::time_point const& now);
 
 private:
-    QueryId  const    _qId{0}; //< queryId from czar
-    int      const    _jId{0}; //< jobId from czar
-    int      const    _attemptCount{0}; // attemptCount from czar
+    QueryId  const    _qId = 0; ///< queryId from czar
+    int      const    _jId = 0; ///< jobId from czar
+    int      const    _attemptCount = 0; // attemptCount from czar
     std::string const _idStr{QueryIdHelper::makeIdStr(0, 0, true)}; // < for logging only
```

Comment:
Would the assignment operator work here to initialize the member like you did for 3 primitive data members above?